client/src/components/Submenu/index.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Initialises the Submenus within the primary Wagtail menu (excluding the Explorer menu)
+ */
+
+const initSubmenus = () => {
+  const primaryNavContainer = document.querySelector('[data-nav-primary]');
+
+  if (!primaryNavContainer) {
+    return;
+  }
+
+  const subMenus = document.querySelectorAll('[data-nav-primary-submenu-trigger]');
+  const activeClass = 'submenu-active';
+
+  subMenus.forEach(subMenu => {
```

Comment:
@jonnyscholes - this breaks in IE 11.
I have asked in Slack but assuming we do still support IE 11, this code breaks badly (screenshot attached)

https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach

![VirtualBox_MSEdge - Win10_23_05_2019_18_24_28](https://user-images.githubusercontent.com/1396140/58237499-91674800-7d88-11e9-8cf1-5061677eaa9f.png)

Reply:
Ah thanks. What do you think about polyfilling IE11 holistically using `@babel/polyfill`? Would mean this issue would come up far less. It will increase the bundle size slightly, but given this is a backend where the full bundle can be cached for the whole admin (and that soonish we intend to load features via code splitting where possible) I think this is ok.

Reply:
For this specific instance I would suggest to convert the NodeList to an array, then you can iterate over it with array methods as usual. Generally I would avoid using iteration methods on NodeLists because it's quite easy to get them confused with arrays and then to have unexpected results.

So in this case this would be:

```js
const subMenus = [].slice.call(document.querySelectorAll('[data-nav-primary-submenu-trigger]'));
```

… or this same logic wrapped in a util function.

Reply:
(as a side note I don't think `@babel/polyfill` includes polyfills for missing DOM methods like this that are specific to browsers, I think it's only polyfilling the JS stdlib)

Reply:
Sure, since we're able to use ES6 features here I'd prefer to use `[...submenus]` though to avoid the need to pull out the verbosity of `[].slice.call(` into a util.

RE `@babel/polyfill` looks like they now advise the use of `core-js` directly now (which afaik does cover some but not all dom apis...). Will look into that in a separate PR :)

Reply:
This should now be resolved. I've not gone for a polyfill and instead cast to an Array as @thibaudcolas advised :)